complex.c

```diff
@@ -1492,21 +1498,28 @@ nucomp_denominator(VALUE self)
 
 /*
  * call-seq:
- *    cmp.numerator  ->  numeric
+ *   numerator -> numeric
+ *
+ * Returns the numerator of +self+:
+ *
+ *   c = Complex(Rational(2, 3), Rational(3, 4)) # => ((2/3)+(3/4)*i)
+ *   c.numerator                                 # => (8+9i)
  *
- * Returns the numerator.
+ * This result is computed thus:
  *
- *        1   2       3+4i  <-  numerator
- *        - + -i  ->  ----
- *        2   3        6    <-  denominator
+ *   # Numerators and denominators for c.real and c.imag.
+ *   c.real.numerator   # => 2
+ *   c.real.denominator # => 3
+ *   c.imag.numerator   # => 3
+ *   c.imag.denominator # => 4
+ *   # Real and imaginary parts for new Complex c.numerator.
+ *   result_real = c.real.numerator * c.imag.denominator # => 8
+ *   result_imag = c.imag.numerator * c.real.denominator # => 9
+ *   # New Complex c.numerator (Q.E.D.).
+ *   result = Complex(result_real, result_imag) # => (8+9i)
+ *   c.numerator                                # => (8+9i)
  *
```

Comment:
I find this example to be difficult to understand. Can we explain using words instead? I think the numerator is the Complex created from the numerators of the real and imaginary parts after converting them to the least common denominator.

Reply:
Revised.